### PR TITLE
[Doctest] Add configuration_convbert.py

### PR DESCRIPTION
**Pull request by AymenBer99**
Add configuration_convbert.py to utils/documentation_tests.txt for doctest.

Based on #19487 

@ydshieh  could you please check it?
Thanks :)

### CODE REVIEW

File: src/transformers/models/convbert/configuration_convbert.py [hunk 1]
```diff
@@ -85,11 +85,11 @@ class ConvBertConfig(PretrainedConfig):
     Example:
 
     ```python
-    >>> from transformers import ConvBertModel, ConvBertConfig
+    >>> from transformers import ConvBertConfig, ConvBertModel
 
     >>> # Initializing a ConvBERT convbert-base-uncased style configuration
     >>> configuration = ConvBertConfig()
-    >>> # Initializing a model from the convbert-base-uncased style configuration
+    >>> # Initializing a model (with random weights) from the convbert-base-uncased style configuration
```
Comment (ydshieh) [inline src/transformers/models/convbert/configuration_convbert.py:92]: Could we add a few new empty lines in this doc example. See for example the BertConfig file 🙏

Comment (AymenBer99) [reply src/transformers/models/convbert/configuration_convbert.py:93]: @ydshieh Added the empty lines. Should be good to go now

**Comment by ydshieh** [comment]
Thank you again, @AymenBer99 🚀

**Review by ydshieh — APPROVED** [review]
Cool!

**Pull request merged**